capture-sdk/sdk/src/main/java/net/gini/android/capture/GiniCapture.java

```diff
@@ -154,123 +159,139 @@ public static synchronized boolean hasInstance() {
         }
         return new Builder();
     }
 
     /**
      * Configure and create a new instance using the returned {@link Builder}.
      *
-     * @return a new {@link Builder}
      * @param context Android context
+     * @return a new {@link Builder}
      */
     @NonNull
     public static synchronized Builder newInstance(final Context context) {
         if (sInstance != null) {
             if (sInstance.mNetworkRequestsManager != null) {
                 sInstance.mNetworkRequestsManager.cleanup();
             }
-            doActualCleanUp(context);
+            cleanup(context);
         }
         return new Builder();
     }
+
     /**
-     * Destroys the {@link GiniCapture} instance and frees up used resources.
+     * Provides transfer summary to Gini.
      *
      * <p>Please provide the required extraction feedback to improve the future extraction accuracy.
      * Please follow the recommendations below:
      *
      * <ul>
+     *     <li>Please call this method before calling the {@link #cleanup(Context)} method</li>
      *     <li>Please provide values for all necessary fields, including those that were not extracted.</li>
      *     <li>Provide the final data approved by the user (and not the initially extracted only).</li>
-     *     <li>Do cleanup after TAN verification.to clean up and provide the extraction values the user has used.</li>
      * </ul>
      *
-     * @param context Android context
      * @param paymentRecipient payment receiver
      * @param paymentReference ID based on Client ID (Kundennummer) and invoice ID (Rechnungsnummer)
-     * @param paymentPurpose statement what this payment is for
-     * @param iban international bank account
-     * @param bic bank identification code
-     * @param amount accepts extracted amount and currency
+     * @param paymentPurpose   statement what this payment is for
+     * @param iban             international bank account
+     * @param bic              bank identification code
+     * @param amount           accepts extracted amount and currency
      */
-    public static synchronized void cleanup(@NonNull final Context context,
-                                            @NonNull final String paymentRecipient,
-                                            @NonNull final String paymentReference,
-                                            @NonNull final String paymentPurpose,
-                                            @NonNull final String iban,
-                                            @NonNull final String bic,
-                                            @NonNull final Amount amount) {
+    public static synchronized void transferSummary(@NonNull final String paymentRecipient, @NonNull final String paymentReference, @NonNull final String paymentPurpose, @NonNull final String iban, @NonNull final String bic, @NonNull final Amount amount) {
```

Comment:
[Request] We should start the method name with a verb to make it clearer what the method does:
```suggestion
    public static synchronized void sendTransferSummary(@NonNull final String paymentRecipient, @NonNull final String paymentReference, @NonNull final String paymentPurpose, @NonNull final String iban, @NonNull final String bic, @NonNull final Amount amount) {
```